src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -131,7 +128,7 @@ public ProcessResult updateCase(
                     existingCase.getId(),
                     exceptionRecord.id
                 );
-                return new ProcessResult(updateResponse.warnings, emptyList());
+                return Optional.of(ErrorsAndWarnings.withErrorsAndWarnings(emptyList(), updateResponse.warnings));
```

Comment:
`ErrorsAndWarnings.withErrorsAndWarnings` can we do static imports here and below ?